target/linux/generic/config-6.1

```diff
@@ -3741,6 +3763,7 @@ CONFIG_MMU_GATHER_RCU_TABLE_FREE=y
 CONFIG_MMU_GATHER_TABLE_FREE=y
 CONFIG_MODPROBE_PATH="/sbin/modprobe"
 CONFIG_MODULES=y
+# CONFIG_MODULE_ALLOW_BTF_MISMATCH is not set
```

Comment:
this should be handled by c07038da27cefa5a93e433909b9aca594386ddc1 already

Reply:
Ah, yes, that was committed after my series so no surprise I missed it. Thanks, will drop that and rebase again.